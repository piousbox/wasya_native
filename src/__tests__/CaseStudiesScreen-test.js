import React from 'react'
import { createStackNavigator, createAppContainer, } from 'react-navigation'

import renderer from 'react-test-renderer'
global.navigator = {
  userAgent: 'node.js'
}

import Component from '../CaseStudiesScreen'

test('renders correctly', () => {
  const tree = renderer.create(<Component navigation={{}} />).toJSON();
  expect(tree).toMatchSnapshot();
})