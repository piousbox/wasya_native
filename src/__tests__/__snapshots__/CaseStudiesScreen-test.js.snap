// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Object {
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "right": 250,
        },
      ]
    }
  >
    <RCTScrollView
      scrollToTop={false}
      style={
        Object {
          "backgroundColor": "white",
          "flex": 1,
          "height": 1334,
          "padding": 20,
          "width": 750,
        }
      }
    >
      <View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "fontSize": 16,
              "fontWeight": "300",
              "padding": 12,
            }
          }
        >
          Home
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "fontSize": 16,
              "fontWeight": "300",
              "padding": 12,
            }
          }
        >
          Our Process
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "fontSize": 16,
              "fontWeight": "300",
              "padding": 12,
            }
          }
        >
          Our Team
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "fontSize": 16,
              "fontWeight": "300",
              "padding": 12,
            }
          }
        >
          Our Clients
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "fontSize": 16,
              "fontWeight": "300",
              "padding": 12,
            }
          }
        >
          Case Studies
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "fontSize": 16,
              "fontWeight": "300",
              "padding": 12,
            }
          }
        >
          Contact Us
        </Text>
      </View>
    </RCTScrollView>
  </View>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
        "height": 1334,
        "left": 0,
        "overflow": "hidden",
        "position": "absolute",
        "top": 0,
        "transform": Array [
          Object {
            "translateX": 0,
          },
        ],
        "width": 750,
      }
    }
  >
    <RCTScrollView>
      <View>
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#476DC5",
                "borderBottomColor": "#f2f2f2",
                "borderBottomWidth": 1,
                "height": 70,
                "padding": 15,
              },
              undefined,
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "flex-end",
                  "flex": 1,
                  "flexDirection": "row",
                  "justifyContent": "space-between",
                },
                undefined,
              ]
            }
          >
            <View
              accessible={true}
              isTVSelectable={true}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Array [
                  false,
                  false,
                  false,
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "transparent",
                    "justifyContent": "center",
                  },
                  undefined,
                ]
              }
            >
              <Text
                accessible={true}
                allowFontScaling={false}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#fff",
                      "fontSize": 24,
                    },
                    Array [
                      Object {
                        "backgroundColor": "transparent",
                      },
                      undefined,
                    ],
                    Object {
                      "fontFamily": "Material Icons",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
              >
                
              </Text>
            </View>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              numberOfLines={1}
              onPress={[Function]}
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Object {
                    "color": "#fff",
                  },
                ]
              }
            >
              Wasya Co
            </Text>
            <View
              style={
                Object {
                  "height": 24,
                  "width": 24,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "backgroundColor": "#F5FCFF",
              "flex": 1,
              "flexDirection": "column",
              "justifyContent": "flex-start",
              "minHeight": 1334,
              "padding": 10,
            }
          }
        >
          <View
            style={Object {}}
          >
            <View
              style={
                Object {
                  "flex": 1,
                  "flexDirection": "column",
                }
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Object {
                    "fontSize": 20,
                    "textAlign": "center",
                  }
                }
              >
                Case Studies
              </Text>
              <View
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                    "justifyContent": "center",
                  }
                }
              >
                <View
                  style={
                    Object {
                      "borderBottomColor": "green",
                      "borderBottomWidth": 6,
                      "height": "60%",
                      "width": "10%",
                    }
                  }
                />
                <View
                  style={
                    Object {
                      "width": 24,
                    }
                  }
                >
                  <View
                    style={
                      Array [
                        false,
                        false,
                        false,
                        Object {
                          "alignItems": "center",
                          "backgroundColor": "transparent",
                          "justifyContent": "center",
                        },
                        undefined,
                      ]
                    }
                    underlayColor="white"
                  >
                    <Text
                      accessible={true}
                      allowFontScaling={false}
                      ellipsizeMode="tail"
                      style={
                        Array [
                          Object {
                            "color": "green",
                            "fontSize": 24,
                          },
                          Array [
                            Object {
                              "backgroundColor": "transparent",
                            },
                            undefined,
                          ],
                          Object {
                            "fontFamily": "Material Icons",
                            "fontStyle": "normal",
                            "fontWeight": "normal",
                          },
                          Object {},
                        ]
                      }
                    >
                      
                    </Text>
                  </View>
                </View>
                <View
                  style={
                    Object {
                      "borderBottomColor": "green",
                      "borderBottomWidth": 6,
                      "height": "60%",
                      "width": "10%",
                    }
                  }
                />
              </View>
            </View>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</View>
`;
